kubernetes/browser.yaml

```diff
@@ -25,6 +26,10 @@ spec:
         - mountPath: /config/
           name: configs
           readOnly: true
+        resources:
+          requests:
+            cpu: 1
```

Comment:
That's a lot, right?

I don't think we provide any resource requests/limits right now, because we have no idea at what scale people are going to use clusters like these.